packages/web/src/utils/number-utils.spec.ts

```diff
@@ -33,10 +33,10 @@ describe("mathSybmolAbsFormat", () => {
 });
 
 describe("The toUnitFormat function returns a value represented in numeric units and up to two decimal places.", () => {
-  test("12345.6789 to $12.35K", () => {
-    const num = 12345.6789;
-    expect(toUnitFormat(num, true, true)).toBe("$12.35K");
-  });
+  // test("12345.6789 to $12.35K", () => {
+  //   const num = 12345.6789;
+  //   expect(toUnitFormat(num, true, true)).toBe("$12.35K");
+  // });
```

Comment:
Fix the results of the test case.

Reply:
I just comment it out fỏ testing , repopened it.